### PR TITLE
Feature/preliminary structure

### PR DESCRIPTION
**Pull request by alpaka-alpachino**
Service skeleton includes .gitignore files in order to push empty directories. Files .gitignore also include short comment with list of expected files in that directory. Would be glad to get your review.

### CODE REVIEW

**Review by maksymbilokur — APPROVED** [review]

**Review by oklymchuk — APPROVED** [review]

**Review by i3odja — DISMISSED** [review]

**Review by oklymchuk — APPROVED** [review]

**Review by crnopster — DISMISSED** [review]

**Review by radikh — CHANGES_REQUESTED** [review]
You don't need gitignore file for each directory,  it's enough to have it in root.
go.mod and go.sum files should not be empty. Use ``` go mod init``` command to generate them.
I would suggest name config for package configs.
Package filter will be better to be situated in the root of the project.

**Review by maksymbilokur — APPROVED** [review]

**Review by oklymchuk — APPROVED** [review]

**Review by maksymbilokur — DISMISSED** [review]

File: build/go.mod [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module github.com/lvl484/positioning-filter/positioning-filter/build
```
Comment (dmitriyminer) [inline build/go.mod]: What do you think about renaming module name to 'github.com/lvl484/positioning-filter'?

Comment (alpaka-alpachino) [reply build/go.mod]: Yep, have changed that)

File: build/go.mod [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module 'github.com/lvl484/positioning-filter'
```
Comment (radikh) [inline build/go.mod]: It would be better to hold mod and sum files in the root of the repo

**Review by radikh — COMMENTED** [review]
Regarding package comments, try to align them as described in a doc: https://blog.golang.org/godoc-documenting-go-code

**Comment by alpaka-alpachino** [comment]
> Regarding package comments, try to align them as described in a doc: https://blog.golang.org/godoc-documenting-go-code

Thanks) Suppose done well. Or should I detalise it more?

File: matcher/position.go [hunk 3]
```diff
@@ -1,3 +1,3 @@
-//Position message model
-
+// Package matcher provides matching positioning messages to a set of filtering rules
+// and pereating messages to a topic in case of they are matched.
```
Comment (radikh) [inline matcher/position.go]: Typo here `pereating`.

File: kafka/topics.go [hunk 4]
```diff
@@ -1,3 +1,3 @@
-//Topics, should be include to kafka: positions, matched-positions.
-
+// Package kafka provides kafka connection.
+// Topics, should be include: positions, matched-positions.
```
Comment (radikh) [inline kafka/topics.go]: This row is about external stuff. Remove it.

File: web/endpoint.go [hunk 5]
```diff
@@ -1,3 +1,4 @@
+// Package web provides managing HTTP protocols.
 // Configurate endpoints to interact with other tools of the project.
-
+// Realise CRUD threw filters.
```
Comment (radikh) [inline web/endpoint.go]: I'm not really sure what does it mean.

File: matcher/position.go (hunk 3 above)
Comment (alpaka-alpachino) [reply matcher/position.go]: My inattention, fixed

File: kafka/topics.go (hunk 4 above)
Comment (alpaka-alpachino) [reply kafka/topics.go]: Yep

File: web/endpoint.go (hunk 5 above)
Comment (alpaka-alpachino) [reply web/endpoint.go]: Well, now I see that it wasn't logical for me even to write that comment, cause from the structure of the program it`is clear that there expected http methods. So I deleted it :)

**Review by oklymchuk — APPROVED** [review]

**Review by oklymchuk — DISMISSED** [review]

**Review by dmitriyminer — APPROVED** [review]

**Review by oklymchuk — APPROVED** [review]

**Review by crnopster — APPROVED** [review]

**Review by maksymbilokur — APPROVED** [review]

**Review by radikh — APPROVED** [review]

**Pull request merged**